docs/individual_sheets.md

```diff
@@ -1359,3 +1417,476 @@ Define references to be associated with properties, etc. as citations.
+| pick from list of <a href="https://criptapp.org/keys/condition-key/" target="_blank">conditions</a> | condition | no       | record your values                                                                                           |
+
+<br>
+
+<table>
+  <tr>
+    <td class="row-1">
+      <u class="row-1">relation</u> 
+    </td>
+    <td class="row-1">
+      <u class="row-1">attribute</u> 
+    </td>
+    <td class="row-1">
+      <u class="row-1">attribute</u> 
+    </td>
+    <td class="row-1">
+      <u class="row-1">attribute</u> 
+    </td>
+    <td class="row-1">
+      <u class="row-1">attribute</u> 
+    </td>
+    <td class="row-1">
+      <u class="row-1">condition</u> 
+    </td>
+
+  </tr>
+  <tr>
+    <td class="row-2">
+      *experiment
+      <br> <span style="font-size: 0.7rem; font-style: italic">(Required)</span>
+    </td>
+    <td class="row-2">
+      *name 
+      <br> <span style="font-size: 0.7rem; font-style: italic">(Required)</span>
+    </td>
+    <td class="row-2">
+      *type 
+      <br> <span style="font-size: 0.7rem; font-style: italic">(Required)</span>
+    </td>
+    <td class="row-2">
+      software_configuration
+      <br> <span style="font-size: 0.7rem; font-style: italic">(optional)</span>
+    </td> 
+    <td class="row-2">
+      notes
+      <br> <span style="font-size: 0.7rem; font-style: italic">(optional)</span>
+    </td>
+    <td class="row-2">
+      <a href="https://criptapp.org/keys/condition-key/" target="_blank">conditions</a>
+      <br> <span style="font-size: 0.7rem; font-style: italic">(optional)</span>
+    </td>
+
+  </tr>
+  <tr>
+    <th class="row-3">
+      <div></div>
+    </th>
+    <th class="row-3">
+      <div></div>
+    </th>
+    <th class="row-3">
+      <div></div>
+    </th>
+    <th class="row-3">
+      <div></div>
+    </th>
+    <th class="row-3">
+      <div></div>
+    </th>
+    <th class="row-3-in-table">
+      <div>units</div>
+    </th>
+
+  </tr>
+  <tr>
+    <td class="row-4 row-4-required-optional-label">
+      value from *name column of experiment sheet
+    </td>
+    <td class="row-4 row-4-required-optional-label">
+      your unique computation name
+    </td>
+    <td class="row-4 row-4-required-optional-label">
+      <a href="https://criptapp.org/keys/computation-type/" target="_blank">Computation Type</a>
+    </td>
+    <td class="row-4 row-4-required-optional-label">
+      name of software_configuration
+    </td>
+    <td class="row-4 row-4-required-optional-label">
+      your notes
+    </td>
+    <td class="row-4 row-4-required-optional-label">
+      your values
+    </td>
+
+  </tr>
+</table>
+
+<details>
+<summary>Notes on multiple software_configurations</summary>
+Use the Id syntax described in <a href="excel_rows.md" target=_blank>Structure of Excel Sheets </a> to have multiple software_configurations in your Computation node
```

Comment:
I would give an example here because chances are the user probably skipped a bunch of pages and when they get to here and are looking at it they'll get confused and have a hard time figuring it out